pkg/agent/route/route_linux.go

```diff
@@ -417,6 +421,41 @@ func (c *Client) syncRoute() error {
 	return nil
 }
 
+func (c *Client) syncNeighbor() error {
+	neighborList, err := c.netlink.NeighList(c.nodeConfig.GatewayConfig.LinkIndex, netlink.FAMILY_ALL)
```

Comment:
maybe add a comment to indicate that Antrea only adds neighbors on the gateway device (which is why we are only listing neighbors for that device here)

Reply:
Done.